packages/lodestar/src/db/api/beacon/repositories/attestation.ts

```diff
@@ -10,10 +12,31 @@ import {IDatabaseController, Bucket, Repository} from "@chainsafe/lodestar-db";
  * Removed when included on chain or old
  */
 export class AttestationRepository extends Repository<Uint8Array, phase0.Attestation> {
+  // For each slot, index attestation roots by attestation data root hex
+  // do batch remove per finalized checkpoint
+  private dataRootIndex = new Map<Slot, Map<string, string[]>>();
```

Comment:
If this map is stored in memory but the attestations to disk, if the node restarts will it loose access to the stored attestations?

Reply:
this index is used for aggregator and its per slot. If we restart we need to index the current slot going forward, not the past slot. So I think we don't need to persist it.